src/BlockListener.php

```diff
@@ -19,4 +19,31 @@ public function type(): int
     {
         return self::TYPE_BLOCK;
     }
+
+    /**
+     * @param Pick $pick
```

Comment:
A phpdoc should also contain description of what this function does, take a look at all the other methods. The more you explain and write, the better other will understand what it does and why its needed. At first sight it might be silly, but for others (or for me taking a look at this function after 1 year) it will make sense.